### DISCUSSION TITLE
Unable to use "GenVertexArray" method in "Avalonia.OpenGL.GlInterface"

### DISCUSSION THREAD

**Discussion by qqJoker** [AvaloniaUI/Avalonia#10737] [Q&A]
I now have a function that needs to use OpenGL to render pictures under Linux, but when I use the Avalonia template to create a project, I find that it is not the same as on GitHub. In the "**Avalonia.OpenGL.GlInterface**" class, I did not find "**GenVertexArray**" method, and the "**OpenGlControlBase**" class is not the same. Currently the Avalonia version I am using is 0.10.18.

**Comment by jp2masa** [comment]
https://github.com/AvaloniaUI/Avalonia/discussions/10646#discussioncomment-5290679
